eskipfile/watch_test.go

```diff
@@ -49,19 +51,42 @@ func createFileWith(content string, t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer os.Remove(tmpfile.Name())
```

Comment:
removing this is a file descriptor leak

Reply:
Is it still a leak if I call os.Remove in the moveFile function?

Reply:
Ah I see, if something fails in moveFile then Remove won't be called :+1: